### PR TITLE
Prepare/1.5.2

### PR DESCRIPTION
**Pull request by bertrand-lorentz**
Backporting updates in translations and view templates from SDK 1.6.0 to prepare SDK 1.5.2.

No field or node have been added in SDK 1.6.0, so the view templates are almost equal to what is in 1.6.0, apart from a few commented out lines.

### CODE REVIEW

File: translations/code_de.xml [hunk 1]
```diff
@@ -9985,14 +9986,12 @@
 <entry key="code|name|currency.SIT">Slovenischer Tolar</entry>
 <entry key="code|name|currency.SKK">Slowakische Krone</entry>
 <entry key="code|name|currency.SLL">Leone</entry>
-<entry key="code|name|currency.SML">San Marino-Liren</entry>
```
Comment (rousso) [inline translations/code_de.xml:9988]: The translation for this code has been removed. Hoever, the code exists in the .gc file. I think we should not remove transaltions when patching.

File: translations/code_de.xml [hunk 2]
```diff
@@ -9985,14 +9986,12 @@
...
 <entry key="code|name|currency.SOS">Somalia-Schilling</entry>
-<entry key="code|name|currency.SQS">Somaliland-Schillingen</entry>
+<entry key="code|name|currency.SQS">Somaliland-Schilling</entry>
 <entry key="code|name|currency.SRD">Suriname-Dollar</entry>
-<entry key="code|name|currency.SRG">Suriname-Gulden</entry>
```
Comment (rousso) [inline translations/code_de.xml:9992]: Same here: Code translation was removed.

File: view-templates/34.efx [hunk 3]
```diff
@@ -283,6 +400,9 @@
 		{BT-03-notice} #{name|BT-03}: #{BT-03-notice}  // Form type
 		{BT-02-notice} #{name|BT-02}: #{BT-02-notice} // Notice type
 		{BT-05(a)-notice} #{name|BT-05}: ${BT-05(a)-notice} ${BT-05(b)-notice} // Notice dispatch date and time
+			// {BT-197(BT-05)-Notice} #{name|BT-197(BT-05)-Notice}: #{BT-197(BT-05)-Notice} // Unpublished Justification Code
```
Comment (rousso) [inline view-templates/34.efx:403]: Normally these commented lines should have been marked as draft. @pdonohoe can verify. The EFX toolkit handles the comments correctly but maybe I would prefer to remove these lines from teh patch.

Comment (bertrand-lorentz) [reply view-templates/34.efx:403]: I remember removing those commented lines in other efx files, I guess I missed them here. You can remove them.

**Review by rousso — APPROVED** [review]
I added a commit with a couple of fixes. Looks good to me now.

**Review by pdonohoe — APPROVED** [review]

**Pull request merged**